uobtheatre/productions/models.py

```diff
@@ -445,6 +442,21 @@ def total_capacity(self, seat_group=None):
         response = self.performance_seat_groups.aggregate(Sum("capacity"))
         return response["capacity__sum"] or 0
 
+    def total_capacity(self, seat_group=None):
+        """Total capacity of the Performance.
+
+        The is the total number of seats (Tickets) which can be booked for this
+        performance. This does not take into account any existing Bookings.
+
+        Returns:
+            int: The capacity of the show
+        """
+        if seat_group is not None:
+            return self.total_seat_group_capacity(seat_group=seat_group)
+        if self.capacity:
+            return min(self.capacity, self.total_seat_group_capacity())
+        return self.total_seat_group_capacity()
+
```

Comment:
@JElgar I dont know if this is the best way of solving the issue (#245) but it seems to work

Reply:
I'm confused, probably just being slow but we should chat about this